src/main/sensors/initialisation.c

```diff
@@ -457,10 +457,10 @@ static void detectMag(magSensor_e magHardwareToUse)
     magSensor_e magHardware;
 
 #ifdef USE_MAG_HMC5883
-    static hmc5883Config_t *hmc5883Config = 0;
+    hmc5883Config_t *hmc5883Config = 0;
 
 #ifdef NAZE
-    hmc5883Config_t nazeHmc5883Config;
+    static hmc5883Config_t nazeHmc5883Config;
```

Comment:
Good idea! That actually saves 12 bytes of BSS since the struct becomes const static instead of mutable.